repokid/commands/repo.py

```diff
@@ -148,7 +150,7 @@ def _rollback_role(
     role_name: str,
     config: RepokidConfig,
     hooks: RepokidHooks,
-    selection: int = 0,
+    selection: int = -1,
```

Comment:
Why was this changed to -1, and the condition below changed to selection < 0 ?

Reply:
This was changed because the selections are zero-indexed, so 0 is a valid selection. The bail-out was previously `if not selection`, which would evaluate to `True` for `selection = 0`. Sneaky sneaky.